devel/rapidjson-devel/Portfile

```diff
@@ -37,6 +39,7 @@ patchfiles                      patch-test-CmakeLists.txt.diff \
 depends_build-append            bin:doxygen:doxygen
 
 compiler.cxx_standard   2011
+compiler.fallback       macports-clang-15
```

Comment:
this is not the proper solution for the problem you noticed. Ports should not add "-march=native"; if  a native build is desirable it should be in a variant. So instead add a patchfile that removes this argument.

Reply:
Thanks for reviewing. I now patch CMakeLists.txt instead and it works.

Reply:
when making that change it will result in a different build so you need to increase the "revision" in both (sub)ports

Reply:
OK, I added a `revision 1` line in the rapidjson-devel Portfile.